web/controllers/friendlist_controller.ex

```diff
@@ -0,0 +1,47 @@
+defmodule Entice.Web.FriendlistController do
```

Comment:
I think the term that is also used in guild wars wiki etc. is 'friends-list" (note the 's') so maybe we want to use that consistently?